README.md

```diff
@@ -125,7 +129,7 @@ Strategies are defined as static methods of the Strategy class in `matching.py`
 - Without making actual replies
 - Without checking a posts_replied_to list
 
-`python src/main.py --skip-tracking`
+`python src/main.py --skip-tracking --simulate-replies`
```

Comment:
` --simulate-replies` is unneeded :D 

As noted above, you won't make replies with `python src/main.py --skip-tracking`